src/services/user_api/routes/auth.js

```diff
@@ -0,0 +1,21 @@
+const router = require('express').Router();
+const User = require('../models/user');
+
+// routes "middle-ware"
+router.post('/register', async (req, res) => {
+  const user = new User({
+    name: req.body.name,
+    email: req.body.email,
+    password: req.body.password
+  });
+  try {
+    const savedUser = await user.save();
+    res.send(savedUser);
+  } catch (err) {
+    res.status(400).send(err);
+  }
+});
+
+//router.post('/login');
+
+module.exports = router;
```

Comment:
migrate to the es6 export by default, learn more [here](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export)

Reply:
Incldude unit tests for this functions.

Reply:
> migrate to the es6 export by default, learn more [here](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export)

Done

Reply:
> Incldude unit tests for this functions.

In progress...